tools/gen_addon_readme.py

```diff
@@ -117,29 +118,58 @@ def make_weblate_badge(repo_name, branch, addon_name):
     )
 
 
-def make_repo_badge(repo_name, branch, addon_name):
+def make_repo_badge(org_name, repo_name, branch, addon_name):
     badge_repo_name = repo_name.replace('-', '--')
     return (
-        'https://img.shields.io/badge/github-OCA%2F{badge_repo_name}'
+        'https://img.shields.io/badge/github-{org_name}%2F{badge_repo_name}'
         '-lightgray.png?logo=github'.format(**locals()),
-        'https://github.com/OCA/{repo_name}/tree/'
+        'https://github.com/{org_name}/{repo_name}/tree/'
         '{branch}/{addon_name}'.format(**locals()),
-        'OCA/{repo_name}'.format(**locals()),
+        '{org_name}/{repo_name}'.format(**locals()),
     )
 
 
-def gen_one_addon_readme(repo_name, branch, addon_name, addon_dir, manifest):
+def generate_fragment(org_name, repo_name, branch, addon_name, file):
+    fragment_lines = file.readlines()
+    if not fragment_lines:
+        return False
+
+    # Replace relative path by absolute path for figures
+    module_url = "https://raw.githubusercontent.com/{org_name}/{repo_name}"\
+        "/{branch}/{addon_name}/".format(**locals())
+    for index, fragment_line in enumerate(fragment_lines):
+        if fragment_line.startswith('.. figure::'):
```

Comment:
@legalsylvain What if the figure is in a indented block? Should we use a regular expression to replace the path what ever its position into the line?

Reply:
I think we should not only search for `figure` but also for `image` as both are quite common
@legalsylvain